include/cell.h

```diff
@@ -117,6 +117,9 @@ class Cell {
   //! \retval insertion_status Return the successful addition of a node
   bool add_neighbour(mpm::Index neighbour_id);
 
+  //! Has neighbours
+  bool has_neighbours() const { return neighbours_.size(); }
```

Comment:
What difference does this function make compared to checking `nneighbours`?

Reply:
@kks32 Yes, I have asked this before and i think we can use `neighbours()` as both are same.

Reply:
Resolved